packages/internal/test-util/src/lib/object-driver/object-driver.module.ts

```diff
@@ -11,13 +13,18 @@ import { ObjectDriverConfig } from './object-driver.config';
  */
 @NgModule()
 export class ObjectDriverModule {
-  static forRoot(config?: ObjectDriverConfig): ModuleWithProviders<ObjectDriverRootModule> {
+  static forRoot(config: Partial<ObjectDriverConfig> = {}): ModuleWithProviders<ObjectDriverRootModule> {
     return {
       ngModule: ObjectDriverRootModule,
       providers: [
         {
           provide: objectDriverConfigToken,
-          useValue: config || {},
+          deps: [lumberjackLogDriverConfigToken],
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackLogDriverConfig => ({
```